main.py

```diff
@@ -1,7 +1,40 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre
+from db.models import Actor
 
 
-def main() -> QuerySet:
-    pass
+def main():
+    # Create genres
+    western = Genre.objects.create(name="Western")
+    action = Genre.objects.create(name="Action")
+    drama = Genre.objects.create(name="Dramma")
+
+    # Create actors and actresses
+    george = Actor.objects.create(first_name="George", last_name="Klooney")
+    keanu = Actor.objects.create(first_name="Kianu", last_name="Reaves")
```

Comment:
There is a typo in the actor's first name 'Kianu'. It should be 'Keanu' to match the update operation on line 29.